setup.py

```diff
@@ -15,8 +15,8 @@
         packages=find_packages(exclude=('test*', 'examples')),
         include_package_data=True,
         install_requires=[
-            'Flask>=0.10',
-            'marshmallow>=2.13',
+            'Flask>=0.10,<1',
```

Comment:
Typo: This should be pinning Flask to `<2`, not `<1`. Compatibility with Flask 1 is tested for in the .travis.yml above.

Reply:
Whoops, good catch